spec/app_spec.rb

```diff
@@ -0,0 +1,20 @@
+ENV['APP_ENV'] = 'test'
+
+require_relative '../app.rb'
+require 'rspec'
+require 'rack/test'
+
+RSpec.configure do |conf|
+  conf.include Rack::Test::Methods
+end
+
+describe 'App' do
+  def app
```

Comment:
Pq precisou disso?

Reply:
o teste falha sem isso
`     Failure/Error: get '/'
     
     NameError:
       undefined local variable or method `app' for #<RSpec::ExampleGroups::App "return response 200 from the / route" (./spec/app_spec.rb:16)>`